python-lib/language_dict.py

```diff
@@ -69,14 +69,14 @@
 """
```

Comment:
Nitpick: can we rename this module to language_support.py?

I have done the same for the text visualization plugin, after realizing it would make it clearer where to find language support info.

Reply:
Done 961a8cf